timeout/timeout.c

```diff
@@ -449,13 +452,14 @@ struct TClock validate_duration(char *timestr) {
  * struct is set.
  */
 int parse_interval(const char *ss, struct TClock *interval) {
-	char s[32];
-	char *afterpoint = NULL,
+	char s[32] = {(char)0},
```

Comment:
`0` is already a valid `char`.

Reply:
Just wanted to make this more explicit.
Could this cause any problem or the compiler will just replace this with a static value anyway?